### PR TITLE
Update CHANGELOG.md for #235

### PR DESCRIPTION
**Pull request by kamicut**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added 
+- "Reset filters" button for campaigns
+
+### Fixed
+- Campaign filters retains state when navigating away from the campaigns search
```
Comment (alyssadelaine) [inline CHANGELOG.md]:
```suggestion
- State retains page number and filters when navigating to and from individual campaign cards
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added 
+- "Reset filters" button for campaigns
```
Comment (alyssadelaine) [inline CHANGELOG.md]:
```suggestion
- "Reset filters" button clears filter options on the Campaigns page
```

File: CHANGELOG.md (hunk 1 above)
Comment (kamicut) [reply CHANGELOG.md]: Yeah but this has to be specific to campaigns because there are filters on the users page that don't have this logic.

Comment (alyssadelaine) [reply CHANGELOG.md]: @kamicut updated

**Pull request merged**